notification/src/config/init/ensure-interface-interaction-custom-type.js

```diff
@@ -1,11 +1,24 @@
-const logger = require('../../utils/logger').getLogger()
+const mainLogger = require('../../utils/logger').getLogger()
 
 const interfaceInteractionType = require('../../../resources/payment-interface-interaction-type.json')
-const config = require('../config')()
+const config = require('../config')
+const ctp = require('../../utils/ctp')
 
-async function ensureInterfaceInteractionCustomType(ctpClient) {
+async function ensureInterfaceInteractionCustomTypeForAllProjects() {
+  const commercetoolsProjectKeys = config.getAllCtpProjectKeys()
+  for (const commercetoolsProjectKey of commercetoolsProjectKeys) {
+    const ctpConfig = config.getCtpConfig(commercetoolsProjectKey)
+    if (ctpConfig.ensureResources)
+      await ensureInterfaceInteractionCustomType(ctpConfig)
+  }
+}
+
+async function ensureInterfaceInteractionCustomType(ctpProjectConfig) {
+  const logger = mainLogger.child({
```

Comment:
Do you want to add this logger context to the extension module?  😅

Reply:
Thanks, I forgot about it 😅  I will add it.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/541/commits/4d0741f4b2d6113bda662ffdf5d7209d4d4faa27